File: Missing-Number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums.size();
        int total = n * (n + 1) / 2;
        int actual = 0;
        for (int num : nums) {
            actual += num;
        }
        return total - actual;
    }
};

File: Missing-Number.py
class Solution(object):
    def missingNumber(self, nums):
        \\\
        :type nums: List[int]
        :rtype: int
        \\\
        n = len(nums)
        total = n * (n + 1) // 2
        actual = sum(nums)
        return total - actual
